test/unit/connect-inject-clusterrole.bats

```diff
@@ -74,110 +74,15 @@ load _helpers
 }
 
```

Comment:
Needing to talk to a local consul client used to be conditional on namespaces or health checks being enabled but now it's always required since the cleanup controller needs it. Since we don't write tests against fields that are always existing in the template it lets us delete a lot of these tests.

Reply:
Nice!